sqlite3/tests/compat/mod.rs

```diff
@@ -244,5 +250,56 @@ mod tests {
                 assert_eq!(sqlite3_close(db), SQLITE_OK);
             }
         }
+
+        #[test]
+        fn test_read_frame() {
+            unsafe {
+                let mut db = ptr::null_mut();
+                assert_eq!(
```

Comment:
This test db should probably be cleared so that the test is idempotent across runs, or use tempfile

```rust
// fn test_read_frame()
assert_eq!(
  sqlite3_open(c"../testing/test_read_frame.db".as_ptr(), &mut db),
  SQLITE_OK
);
```